packages/documentation/src/pages/architecture/site-structure/topology-overview.mdx

```diff
@@ -0,0 +1,64 @@
+# Topology Overview
+
+## www.va.gov technical architecture
+
+This document walks through the basics of the technical architecture that supports the Lighthouse Platform API and www.va.gov. This document starts with simple conceptual descriptions and becomes increasingly detailed, allowing the reader to proceed only as far as they need to.
+
+### Basic Concepts
+
+At the highest level, users make requests to www.va.gov and api.va.gov.  These requests are routed to GovCloud, which can then 1. respond to the requests on its own (such as for static content) or 2. reach into the VA network to call an API to help respond to the request (such as to get a user's prescription information).
+
+![](./images/ConceptualArchitecture.png)
```

Comment:
Couldn't figure out how to get these images to work. We have some relative path images in the forms docs but it gatsby isn't picking these up with the local build.